src/models/room.js

```diff
@@ -1141,6 +1136,25 @@ Room.prototype.addPendingEvent = function(event, txnId) {
 
     this.emit("Room.localEchoUpdated", event, this, null, null);
 };
+// live events are aggregated in the timelineset
```

Comment:
Yeah, I went with non-live because it's also used to reapply a relation when the redaction for it is cancelled. In this case the relation might not just be a local echo anymore, but already have gone through the timelineset addliveevent path before the local echo of the redaction was applied to it. This is why I was hesitant to go with a name that has local echo in it. `_aggregateRelationEvent` seems better indeed but doesn't explain why this is not used for live events, in that one could assume this is the only path that relations get aggregated when reading it.

How about `_aggregateNonLiveRelation` with a better comment was is meant by non-live?

Reply:
Thanks, I think it's clearer now with the updated comment. 😁